tools/coverage/jacoco-common.gradle

```diff
@@ -0,0 +1,111 @@
+apply plugin: 'jacoco'
+
+jacoco {
+    toolVersion = "$versions.jacoco"
+}
+
+tasks.withType(Test) {
+    jacoco.includeNoLocationClasses = true
+}
+
+private static void addPathIfExists(List<String> paths,
+                                    String directoryPath) {
+    if (new File(directoryPath).exists())
+        paths.add(directoryPath)
+}
+
+private static void addFileTreeIfExists(Project project, List<FileTree> fileTrees,
+                                 String directoryPath, List<String> excludeFilter = []) {
+    if (new File(directoryPath).exists())
+        fileTrees.add(project.fileTree(dir: directoryPath, excludes: excludeFilter))
+}
+
+static List<String> findSourceDirectories(Project project, String flavor = "", String buildType = "debug") {

```

Comment:
Here instead of having `Project` as first parameter, could it be only the `projectDir`.
Or you could also assign a variable with the value `project.projectDir`. So you avoid code repetion on lines 28,29,33,34,39,40

Reply:
Sí pero no... jajaja

I'm joking... But I don't know, for now it seems ok, to call this method outside is easier.